src/Book.css

```diff
@@ -1,4 +1,4 @@
-.books {
+.book {
```

Comment:
The same here. If your component is not directly related to precise entity - you'll be able to cal it simply index.css and avoid confusion.